newa/cli.py

```diff
@@ -148,6 +149,11 @@ def main(click_context: click.Context,
         )
     click_context.obj = ctx
 
+    # this is here just to suppress state-dir creation
+    # and further processing in case of 'list' subcommand
+    if 'list' in sys.argv:
+        return
+
```

Comment:
Can we move it after debug logging setup (I can see a valid use case to call it with --debug)?

Reply:
Done. Also preserved set DEBUG level in the `list` subcommand.